fighthealthinsurance/generate_prior_auth.py

```
@@ -148,19 +150,24 @@ async def _generate_single_proposal(
+            substituted_text = await sync_to_async(
+                PriorAuthTextSubstituter.substitute_patient_and_provider_info
+            )(prior_auth, proposal_text)
+
             # Create a unique ID for this proposal
             proposed_id = uuid.uuid4()
 
             # Create and save the proposal in the database, with sqlite this can result in db locked errors.
             try:
-                await self._create_proposal(prior_auth, proposed_id, proposal_text)
+                await self._create_proposal(prior_auth, proposed_id, substituted_text)
             except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)